content/references/protocol-reference/transactions/transaction-types/nftokencanceloffer.md

```diff
@@ -41,8 +40,11 @@ This transaction removes the listed `NFTokenOffer` object from the ledger, if pr
 | `TransactionType` | String    | UInt16            | NFTokenCancelOffer transaction type. The integer identifier is 28. |
 | `TokenOffers`     | Array     | VECTOR256         | An array of IDs of the `NFTokenOffer` objects to cancel (not the IDs of `NFToken` objects, but the IDs of the `NFTokenOffer` objects). Each entry must be a different [object ID](ledger-object-ids.html) of an [NFTokenOffer](nftokenoffer.html) object; the transaction is invalid if the array contains duplicate entries. |
 
-The transaction can succeed even if one or more of the IDs in the `TokenOffers` field do not refer to objects that currently exist in the ledger. (For example, those token offers might already have been taken.) The transaction fails with an error if one of the IDs points to an object that does exist, but is not a [NFTokenOffer](nftokenoffer.html) object.
+The transaction can succeed even if one or more of the IDs in the `TokenOffers` field do not refer to objects that currently exist in the ledger. (For example, those token offers might already have been taken.)
 
+It is important to note that if you inadvertently provide a `NFTokenID` rather than a `NFTokenOfferID`, you might receive a `tesSuccess` response. The reason is that when passed a properly formatted VECTOR256 value that is not found, the system assumes that the NFTokenOffer has already been deleted.
```

Comment:
`NFTokenID` and `NFTokenOfferID` are not official names for data types, so it's confusing to put those in monospace font. But it works if you refer to the "ID" of those types of objects. Also, `tesSUCCESS` is the proper capitalization of the error code in the API. Also, the list is a "VECTOR256" but each item _in_ the list is a "Hash256" type (a raw 256-bit value of any kind—a slight misnomer since they were originally or mostly intended to hold hash values but in some cases hold other types of data). 

```suggestion
It is important to note that if you inadvertently provide a the ID of a `NFToken` object rather than a `NFTokenOffer` object, you might receive a `tesSUCCESS` response. The reason is that when passed a properly formatted ID value that is not found, the system assumes that the `NFTokenOffer` has already been deleted.
```

Reply:
This is fine, but need to delete the "a" in "...inadvertently provide *a* the ID of a...."

Reply:
Edited my suggestion to fix that.